### PR TITLE
Added red neo4j. Configured publish pipelines for red neo4j.

### PR DESCRIPTION
**Pull request by catalin-ilea**

### CODE REVIEW

File: content-ingester-neo4j-red-sidekick@.service [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+After=content-ingester-neo4j-red@%i.service
+
+[Service]
+ExecStart=/bin/sh -c "\
+  etcdctl set ft/services/content-ingester-neo4j-red/healthcheck true; \
+  etcdctl mkdir /ft/services/content-ingester-neo4j-red/servers; \
+  etcdctl set /ft/healthcheck/content-ingester-neo4j-red-%i/path /__health; \
+  while true; do \
+    PORT=`[ $PORT ] && echo $PORT || echo $(/usr/bin/docker port content-ingester-neo4j-red-%i 8080 | cut -d':' -f2)`; \
+    etcdctl set /ft/services/content-ingester-neo4j-red/servers/%i "http://$HOSTNAME:$PORT" --ttl 600; \
```
Comment (Zapadlo) [inline content-ingester-neo4j-red-sidekick@.service]: I'm assuming it works, nevertheless those double quoutes are unescaped...

Comment (catalin-ilea) [inline content-ingester-neo4j-red-sidekick@.service]: I does work, yes

**Comment by Zapadlo** [comment]
+1

**Comment by tosan88** [comment]
:+1:

**Comment by martingartonft** [comment]
Why do our singleton neo4j service files contain %i in them?  I think we can only ever have one of each?

**Pull request merged**